### PR TITLE
fix(Google Login): Prevents uninvited users gaining access (LLC-10)

### PR DESCRIPTION
**Pull request by crazy-grizzly**
https://learningpool.atlassian.net/browse/LLC-10
https://learningpool.atlassian.net/browse/LL-295

### CODE REVIEW

File: ui/src/utils/oauth.js [hunk 1]
```diff
@@ -78,29 +78,80 @@ function getEndpoint(provider) {
 }
 
 export function openPopup(provider) {
+  console.log('openPopup', provider);
```
Comment (ryansmith94) [inline ui/src/utils/oauth.js]: I think we should probably remove this.

File: ui/src/utils/oauth.js [hunk 2]
```diff
@@ -78,29 +78,80 @@ function getEndpoint(provider) {
...
+
+/**
+ * @param {Window} popup
+ * @param {checkForToken~tokenResolve} resolve
+ * @param {checkForToken~tokenReject} reject
+ */
 function checkForToken(popup, resolve, reject) {
-  if (popup.closed) reject({ errors: 'Authentication was cancelled.' });
-  else {
+  console.log('checkForToken');
```
Comment (ryansmith94) [inline ui/src/utils/oauth.js]: Again should probably remove this.

Comment (crazy-grizzly) [reply ui/src/utils/oauth.js]: Ah damn 🙂 Maybe we should setup eslint rule for debug messages?

Comment (ryansmith94) [reply ui/src/utils/oauth.js]: Yeah agreed.

**Pull request merged**